modules/dependencies.nix

```
@@ -9,6 +9,11 @@ let
 
   packages = {
     bat.default = "bat";
+    codeium.default = "codeium";
+    coreutils = {
+      default = "coreutils";
+      example = "pkgs.uutils-coreutils";
```

Conflict with #3180:

```suggestion
      example = "uutils-coreutils";
```